src/mixins/abstractComponent.ts

```diff
@@ -35,6 +36,14 @@ export const AbstractComponent = <T extends Component>(superClass: T): Component
 
     private reactiveController = new ReactiveController(this);
 
+    public firstUpdated(args: any) {
```

Comment:
firstUpdated seems late. Can we do this in the constructor?

Would prefer to reduce chance of FOUC